scripts/runtestsuite/runtestsuite.py

```diff
@@ -117,6 +117,41 @@ def write_rgba_png_file(filepath, rows):
     with open(filepath, 'wb') as file:
         writer.write(file, rows)
 
+def git_installed(directory):
+    temp = 'git -C {0} --version'.format(directory)
+    print(temp)
+    return command_is_valid('git -C {0} --version'.format(directory))
+
+def under_git_control(directory):
+    return command_is_valid('git -C {0} rev-parse --is-inside-work-tree'.format(directory))
+
+def get_git_hash(directory):
+    return command_output('git -C {0} rev-parse HEAD'.format(directory))
+
+def get_git_title(directory):
+    return command_output('git -C {0} rev-parse --abbrev-ref HEAD'.format(directory))
+
+def command_output(command):
+    output = "N/A"
```

Comment:
This is what will be printed when the command fails (e.g. `get_git_hash` fails). I thought it would be more consistent to have it print `N\A` rather than `none`. 

I could have an `if else` statement inside `__write_header` but I thought that would make it unnecessarily messy, do you think it would be a better choice?

Reply:
I think it's a good idea to print "N/A" for the user. But here the method is `command_output` and we don't expect it to improve the output by default.